bindings/node.js/test/kzg.test.ts

```diff
@@ -234,19 +263,29 @@ describe("C-KZG", () => {
   });
 
   describe("edge cases for blobToKzgCommitment", () => {
+    describe("check argument count", () => {
```

Comment:
space out
```suggestion

    describe("check argument count", () => {
```

Reply:
The lint rules prevent this change.  I added spaces between `describe` and `it` blocks where possible and ran lint.  Please verify things look acceptable.